pom.xml

```
@@ -11,7 +11,7 @@
     <url>https://github.com/evis/scalafix-maven-plugin</url>
     <groupId>io.github.evis</groupId>
     <artifactId>scalafix-maven-plugin</artifactId>
-    <version>0.1.3_0.9.5</version>
+    <version>0.1.3_0.9.23</version>
```

```suggestion
    <version>0.1.4_0.9.23</version>
```

Also, would be great to fix versions in README :)

(can do it myself, though)

---

changed the version and updated README